.github/workflows/scripts/pre-publish-hook

```diff
@@ -13,50 +13,6 @@ echo updating $f...
 condition="'\$(GITHUB_ACTIONS)' == 'true'"
 yq -i -o xml -p xml '
   ( del( .Project.ItemGroup[0] | select(.+@Include == "..\docs\**\*") )
-  | .Project.PropertyGroup = [
-      .Project.PropertyGroup,
-      [
-        {
-          "+@Condition": "'"$condition"'",
-          "ContinuousIntegrationBuild": "true",
-          "EmbedUntrackedSources": "true",
-          "IncludeSymbols": "true",
-          "PublishRepositoryUrl": "true",
-          "SymbolPackageFormat": "snupkg"
-        },
-        {
-          "Copyright": "Copyright (c) Shippo 2024",
```

Comment:
This is resolved by pulling from the `author` field in gen.yaml.
Result is in this format: `Copyright (c) <author> <current year>`